tests/integration/test_coalescer.py

```diff
@@ -195,7 +197,7 @@ async def _run(cmd):
         stdout=asyncio.subprocess.PIPE,
         stderr=asyncio.subprocess.PIPE)
 
-    stdout, stderr = await proc.communicate()
+    _, _2 = await proc.communicate()
```

Comment:
I think here you can also write `await proc.communicate()` and the linter doesn't complain

Reply:
True enough, will fix.